pkg/limayaml/limayaml.go

```diff
@@ -7,6 +7,7 @@ import (
 )
 
 type LimaYAML struct {
+	Driver            *Driver           `yaml:"driver,omitempty" json:"driver,omitempty"`
```

Comment:
Should this be called like `VMType` for consistency with `MountType`?

Reply:
Also please create a document like `docs/vmtype.md` or `docs/vz.md` to explain the usage, requirements, and limitations of vz

Reply:
Isn't VMType kind of gives a wrong meaning (Type of VM like linux etc).

How about DriverType ?

Reply:
> Type of VM like linux etc

We only support Linux.
Eventually we may support FreeBSD (if somebody contributes), but that will be probably called "type of guest", not "type of VM"

Reply:
Done the VMType change only for yaml config related places. 
Do let me know if you want to change the word driver everywhere (like driver interface, implementations etc).

Will add doc related things as next step.

Reply:
Thanks, "driver" in the impl is ok